docs/csharp/language-reference/compiler-messages/cs0188.md

```diff
@@ -13,10 +13,8 @@ The 'this' object cannot be used before all of its fields are assigned to
   
  All fields in a `struct` have to be assigned by a constructor before the constructor can call a method in the `struct`.  
   
- If you see this error when trying to initialize a property in a struct constructor, the solution is to change the constructor parameter to specify the backing field instead of the property itself. Auto-implemented properties should be avoided in structs because they have no backing field and therefore cannot be initialized in any way from the constructor.  
```

Comment:
As far as I understand, beginning with C# 6, auto-implemented properties can be initialized in a struct constructor. So, removing bits about auto-implemented properties from this article.